### ISSUE TITLE
compare with codeformer？

### ISSUE THREAD

**Issue by cosmicrealm** [ohayonguy/PMRF#5]
It seem that the evaluation indicators not same with codeformer?  https://arxiv.org/pdf/2206.11253

**Comment by ohayonguy** [comment]
Hi :)
We recomputed ourselves all the metrics for all compared methods. All our evaluation code (which we adopt from previous packages) is provided in our repo, so feel free to compare our evaluation code with that of CodeFormer, and/or reproduce our results.

I'll note that we are not the only one having a discrepancy with the results reported in the CodeFormer paper. For example, see https://arxiv.org/pdf/2312.15736. But this is not problematic, since the evaluation codes for PSNR, SSIM, LPIPS, etc., may vary between packages. I am not sure how CodeFormer evaluated LPIPS, for instance. Unfortunately I can't find the exact evaluation codes in their repo.

I'll also say that we computed FID, KID and Precision w.r.t the entire 70,000 FFHQ images. I believe that CodeFormer computed these metrics w.r.t. 50,000 FFHQ images (generally a larger number is more accurate).

I hope this clarifies things. Thanks!
